Tools/CMake/Pack.cmake

```diff
@@ -0,0 +1,181 @@
+# Pack.cmake packages the JASP binary
+#
+# On Windows,
+#   - We have two bundler, WIX and ZIP; and they can be called by their target name
+#       - `cmake --build . --target wix`
+#       - `cmake --build . --target zip`
+#   - In addition, there are two targets for collecting and recreating junctions that
+#     are being called automatically before the creation of WIX,
+#       - `cmake --build . --target recreate-junctions`
+#       - `cmake --build . --target collect-junctions`
+#
+# On macOS,
+#   - We are using the `create-dmg` script to create and design the DMG. You need to
+#     install it before you can configure JASP. You can download it from Homebrew
+#     using `brew install create-dmg`
+#       - `cmake --build . --target dmg`
+#   - You can also apply the binary for notarisation using the `notarise` command
+#       - `cmake --build . --target notarise`
+#   - Following the successful notarisation, you can staple the DMG using the `staple` target
+#       - `cmake --build . --target staple`
+#
+list(APPEND CMAKE_MESSAGE_CONTEXT Pack)
+
+set(CPACK_PACKAGE_NAME "JASP")
+set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "A Fresh Way to Do Statistics")
+set(CPACK_PACKAGE_HOMEPAGE_URL "https://jasp-stats.org")
+set(CPACK_PACKAGE_VENDOR "University of Amsterdam")
+set(CPACK_PACKAGE_CONTACT "EJ Wagenmakers")
```

Comment:
I really don't know. What I know is that at the moment, the macOS version is signed by Bruno, and the Windows version is signed by UvA. Is that the case? If so, then we probably need to unify this, maybe sign everything with the UvA, or get a certificate from them and sign the macOS version with that as well.


P.S. These are not being used at the moment, I was hoping to have the CPack build the installer, but there are issues on the way of it, so, for now, these guys have no effect but the aforementioned signature problem still holds.

Reply:
> What I know is that at the moment, the macOS version is signed by Bruno, and the Windows version is signed by UvA. Is that the case?

Yes!

Previously they also signed the mac-version but after the notarizing was required they couldn't do it anymore (and I had to modify their script a bit to get it to work) and we stopped bothering them.
However, now that script+notarizing also broke and all your hard work went into making it work again.

We will not get their key though, how hard would it be for them to just sign JASP?

Reply:
Hard! 

Because they need to sign it, and the notarise it and some of the signing is being done by Qt, CMake, etc. If we want to we can explore the idea of setting up a script but that's not going to be easy either since they might need all the dev tools.

Reply:
Nah, I think it is fine as is, nobody has complained about Bruno being the signer yet. Having UvA would look cooler but not important really.